bin/generate_qna_crawl.py

```diff
@@ -0,0 +1,181 @@
+"""
+Script Purpose: This script generates questions based on provided prompts and
+stores the responses as JSON files. It interacts with the AI model to create
+questions and saves the relevant data for each question in a JSON file.
+
+Usage: ./generate_qna_crawl.sh PROMPT_PATH
+
+Parameters: - PROMPT_PATH: Directory containing the API prompt files
+(qna_system_prompt.txt, qna_user_prompt.txt, and JSON template)
+"""
+
+import os
+import json
+from datetime import date
+import argparse
+
+import ailab.db as db
+from ailab.models import openai
+import ailab.db.finesse as finesse
+
+from ailab.db.finesse.test_queries import get_random_crawl
+
+# Constants
+TEST_VERSION = date.today()
```

Comment:
Should be an argument

Reply:
I think you mean the number of question wanted, indeed, this would be better.